config-templates/netapp/netapp-ontap-san/20.07/metadata.json

```
@@ -0,0 +1,50 @@
+{
+  "name": "20.07",
+  "enabled": "true",
+  "description": "Netapp Ontap-SAN Driver (Trident version 20.07)",
```

Where is this description displayed? in the UI & CLI? I think in both places we also display the version value (or in this case the "name" value from this file) so I suggest we remove the version from this description. I will add this suggestion to the other templates.

```suggestion
  "description": "Netapp Ontap-SAN.",
```